zebra/zebra_vty.c

```diff
@@ -1351,26 +1347,23 @@ DEFPY (show_interface_nexthop_group,
 
 DEFPY (show_nexthop_group,
        show_nexthop_group_cmd,
-       "show nexthop-group rib <(0-4294967295)$id|[<ip$v4|ipv6$v6>] [vrf <NAME$vrf_name|all$vrf_all>]>",
```

Comment:
I'm not quite following this: singleton nhes are capturing an afi, although groups use a generic value. why not be able to look at the singletons using this sort of filter?